src/frontend/package.json

```diff
@@ -8,17 +8,22 @@
     "start": "next start"
   },
   "dependencies": {
+    "@emotion/core": "^11.0.0",
+    "@emotion/react": "^11.1.5",
+    "@emotion/styled": "^11.1.5",
     "@types/react-modal": "^3.12.0",
     "axios": "^0.21.1",
     "class-transformer": "0.3.1",
+    "framer-motion": "^4.1.2",
     "next": "10.0.6",
     "react": "17.0.1",
     "react-circular-progressbar": "^2.0.3",
     "react-csv": "^2.0.3",
     "react-dom": "17.0.1",
     "react-input-slider": "^6.0.0",
     "react-modal": "^3.12.1",
-    "swr": "^0.5.4"
+    "swr": "^0.5.4",
+    "toasted-notes": "^3.2.0"
```

Comment:
I think you're missing react-spring potentially, or at least that's causing vercel to error

Reply:
```

19:06:46.630 | info  - Creating an optimized production build...
-- | --
19:06:53.025 | Failed to compile.
19:06:53.026 | ModuleNotFoundError: Module not found: Error: Can't resolve 'react-spring' in '/vercel/workpath0/src/frontend/node_modules/toasted-notes/lib'


```
In case you were wondering what the error message was @nickhil-tekwani